Zad1/src/sql/demo/Main.java

```diff
@@ -0,0 +1,60 @@
+package sql.demo;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.util.Scanner;
+
+public class Main {
+
+
+    public static void main(String[] args) throws Exception{
+        Scanner scanner = new Scanner(System.in);
+        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
+        System.out.println("Введите ширину рамки");
+        int a = scanner.nextInt();
+        System.out.println("Введите длину рамки");
+        int b = scanner.nextInt();
```

Comment:
[ERROR] /Users/geoiva/Desktop/SberJava/students/alena_kochurova/nordic-it-java-q4/Zad1/src/sql/demo/Main.java:16:13: Local variable name 'b' must match pattern '^[a-z][a-z0-9][a-zA-Z0-9]*$'. [LocalVariableName]